### PR TITLE
feat: external events

### PR DESCRIPTION
**Pull request by obi-ewan**
Add existing/old `external-events` repository code to the updated public repository

### CODE REVIEW

File: Anchor.toml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[programs.localnet]
+externalevent = "5qCutonYoeg1aRK31mv4oQYoKdNFMpPaEtDe9nnNQXXf"
+
+[registry]
+url = "https://anchor.projectserum.com"
```
Comment (pawel-betdex) [inline Anchor.toml:5]: is it still project serum?

File: programs/externalevent/src/instructions/update.rs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    let event = &mut ctx.accounts.event;
+    event.participants = participants;
+    Ok(())
+}
+
+pub fn update_start_timestamp(ctx: Context<UpdateEvent>, timestamp: i64) -> Result<()> {
+    let event = &mut ctx.accounts.event;
+    event.start_expected_timestamp = timestamp;
+    Ok(())
+}
```
Comment (pawel-betdex) [inline programs/externalevent/src/instructions/update.rs:39]: file end

File: programs/externalevent/src/lib.rs [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        oracle: String,
+        oracle_reference: String,
+    ) -> Result<()> {
+        instructions::create(ctx, slug, name, event_type, start_expected_timestamp, participants, oracle, oracle_reference)?;
+        Ok(())
+    }
+
+    pub fn update_score(
+        ctx: Context<UpdateEvent>,
+        _slug: String,
```
Comment (pawel-betdex) [inline programs/externalevent/src/lib.rs:37]: ?

Comment (obi-ewan) [reply programs/externalevent/src/lib.rs:37]: this stuff will all get stripped out soon, this is just an initial PR to get the repositories kick started

File: ci/deploy_manager/devnet_history/dev.json [hunk 4]
```diff
@@ -0,0 +1,76 @@
+{
+  "deploys": [
```
Comment (pawel-betdex) [inline ci/deploy_manager/devnet_history/dev.json:2]: i think we stopped ding this on protocol so maybe here gone too?

**Review by pawel-betdex — APPROVED** [review]

**Pull request merged**